clang/lib/Sema/SemaCUDA.cpp

```diff
@@ -241,11 +245,11 @@ SemaCUDA::CUDAVariableTarget SemaCUDA::IdentifyTarget(const VarDecl *Var) {
 // | h  | d  |    HD(y) |     WS(v)  |     N(x)  | ( ) |
 // | h  | g  |    N     |     N      |     N     | (c) |
 // | h  | h  |    N     |     N      |     SS(p) | ( ) |
-// | h  | hd |    HD    |     HD     |     HD    | ( ) |
+// | h  | hd |    SS(u) |     HD     |     HD    | ( ) |
 // | hd | d  |    HD(y) |     SS     |     N(x)  | ( ) |
-// | hd | g  |    SS    |     --     |    --(z)  |(d/a)|
-// | hd | h  |    SS    |     WS     |     SS    | (d) |
-// | hd | hd |    HD    |     HD     |     HD    | (b) |
+// | hd | g  |    N     |     --     |     --(z) |(d/a)|
+// | hd | h  |    N     |     WS     |     HD(s1)| (d) |
+// | hd | hd |    SS    |     HD     |     SS(s2)| (b) |
```

Comment:
I can't find a legend for what the parenthesized values mean in the various columns and I'm not able to intuit their meaning.